README.md

```diff
@@ -35,9 +35,19 @@ pipx ensurepath
 
 Upgrade pipx with `brew update && brew upgrade pipx`.
 
+
 ### On Linux, install via pip (requires pip 19.0 or later)
 
+-  Ubuntu 23.04 or above
+
+```sh
```

Comment:
I think you should adapt your style to the rest of the docs and remove the `sh`.